ldap_auth_provider.py

```diff
@@ -48,6 +48,7 @@
 
 
 class ActiveDirectoryUPNException(Exception):
+    """Raises in case Matrix ID can not be mapped to UPN"""
```

Comment:
it's not a matrix id.

```suggestion
    """Raised in case the user's login credentials cannot be mapped to a UPN"""
```